.env.example

```diff
@@ -11,6 +11,12 @@ ETHEREUM_RPC_URL=
 MEMORY_PAGES_CONTRACT_ADDRESS=
 STARKNET_SOLIDITY_CORE_CONTRACT_ADDRESS=
 
+# Celestia DA Client
+CELESTIA_DA_RPC_URL=http://localhost:8000
+# CELESTIA_DA_AUTH_TOKEN
```

Comment:
is this commented because it's optional? i think the standard is to not passing anything after = (mayeb you can add a comment that it's optional)

Reply:
Yes this is optional.
Noted, wasn't aware about the convention.